File: isem-finanzas-web/src/app/@finanzas/@auditorias/models/requerimiento-asignado.model.ts
export class ResponsableRequerimientoModel {
  id_requerimiento_asignado?: number;
  id_requerimiento?: number;
  id_asignado?: number;
  nombre_asignado?: string;
  fecha_asignacion?: string;
  fecha_cancelacion?: string;
  aceptado?: boolean;
  fecha_atendido_requerimiento?: string;
  atendido_requerimiento?: boolean;
  fecha_atendido_observacion?: string;
  atendido_observacion?: boolean;
  activo?: boolean;
  accion?: number;
}

File: isem-finanzas/common/backend/database/Models/Modelos/RequerimientoAsignadoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Models.Modelos
{
    public class RequerimientoAsignadoModel
    {
        public long? id_requerimiento_asignado { get; set; }
        public long? id_requerimiento { get; set; }
        public long? id_asignado { get; set; }
        public string fecha_asignacion { get; set; }
        public string fecha_cancelacion { get; set; }
        public bool? aceptado { get; set; }
        public string fecha_atendido_requerimiento { get; set; }
        public bool? atendido_requerimiento { get; set; }
        public string fecha_atendido_observacion { get; set; }
        public bool? atendido_observacion { get; set; }
        public bool? activo { get; set; }
        [NotMapped] public int? accion { get; set; }
        [NotMapped] public int tipoOperacion { get; set; }
    }
}
